app-misc/ledit/ledit-2.04.ebuild

```diff
@@ -1,16 +1,16 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=5
 inherit eutils
 
 DESCRIPTION="A line editor to be used with interactive commands"
-SRC_URI="http://pauillac.inria.fr/~ddr/ledit/distrib/src/${P}.tgz"
-HOMEPAGE="http://pauillac.inria.fr/~ddr/ledit/"
+HOMEPAGE="http://pauillac.inria.fr/~ddr/${PN}"
+SRC_URI="${HOMEPAGE}/distrib/src/${P}.tgz"
 
 LICENSE="BSD"
 SLOT="0"
-KEYWORDS="amd64 ppc x86"
+KEYWORDS="~amd64 ~ppc ~x86"
```

Comment:
Would it be fine to not change this line? Stable version doesn't build anyway.

Reply:
No, you can't commit straight to stable. However I think it's fine to stabilize the updated version a bit faster. 

Would you like to become the maintainer for this package to proceed with stabilization & removing old?

You also can't remove the old ebuild before newer one is stabilized. You should reset your branch and do like this: 
```
cd app-misc/ledit
cp ledit-2.03.ebuild ledit-2.04.ebuild
repoman manifest
<edit whatever you need with 2.04>
git add Manifest ledit-2.04.ebuild
repoman ci
git push -f origin yourbranch
```